ecosystem/indexer-grpc/indexer-grpc-manager/src/config.rs

```diff
@@ -1,18 +1,34 @@
 // Copyright © Aptos Foundation
 // SPDX-License-Identifier: Apache-2.0
 
+use crate::grpc_manager::GrpcManager;
 use anyhow::Result;
 use aptos_indexer_grpc_server_framework::RunnableConfig;
 use serde::{Deserialize, Serialize};
+use std::net::SocketAddr;
+use tokio::sync::OnceCell;
+
+static GRPC_MANAGER: OnceCell<GrpcManager> = OnceCell::const_new();
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub(crate) struct ServiceConfig {
```

Comment:
do we want to have a custom health check endpoint so that k8s can kick-in to restart the pod if error happens.

Reply:
Good point. Will come up with something later.